src/core/include/openvino/op/roi_align.hpp

```diff
@@ -84,6 +84,71 @@ class OPENVINO_API ROIAlign : public Op {
     PoolingMode m_mode;
 };
 }  // namespace v3
+
+namespace v9 {
+class OPENVINO_API ROIAlign : public Op {
+public:
+    OPENVINO_OP("ROIAlign", "opset9");
+    enum class PoolingMode { AVG, MAX };
+    enum class AlignedMode { ASYMMETRIC, TF_HALF_PIXEL_FOR_NN, HALF_PIXEL };
```

Comment:
nitpick: we could use the same order of aligned modes options in the spec and in the code

Reply:
this order maybe better, have modified spec to align